### PR TITLE
Cucumber

### PR DESCRIPTION
**Pull request by jaffamonkey**
Current CI tests that are going green.  Steps to add, but good point to get this back into master.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -101,7 +101,7 @@ install:
 # Stamp the code with the build number
 #-----------------------------------------------------------------------------
 
-  - 'echo "{ \"build\" : \"${TRAVIS_BUILD_NUMBER}\", \"tag\":\"${TRAVIS_TAG}\", \"time\":\"$(date -u)\"}" > web/build_version.txt'
+  - 'echo "#${TRAVIS_BUILD_NUMBER} ${TRAVIS_TAG}: $(date -u)" > web/build_version.txt'
```
Comment (stemount) [inline .travis.yml]: @jaffamonkey quick one - this is supposed to be the line above

File: .travis.yml [hunk 2]
```diff
@@ -178,4 +177,4 @@ addons:
...
+    secure: oRuAo84heWUp6MVcM22dx/FZUV5HUVvsegEDaiVggQmNo2zYzNQhgAY7P9GpnaPWo53inCoFfwrwn9V+sj2uwUwrtWQg71qCTU3FIZicqYBB44rp+7/YdpuNqCYtbbY50UJpsZtqRIffWB4iS0YZ2bd+SkvsJskzRzbHC9qRjrSq0Hmb1DuxQCiE+M9TM7n4phY1dPTxTm1g1WwShr1zKXTfhDDK2k9r9kQGpJFP5rrE5cch0kBPJP29rgijeJq5uFS99+QyiVRJcqStXYl2JvKXTMS3sVk+OgSHeeRaaY0dbQSPcn4mTgcQKMVpfAUXVD6mUlkj4WYDXjdQSZdbgVvsX5tQg/6kG52wXgB+vDsmwKoBcTZLndNUQc9nCA02A+WVLyru318JmxwtaLo7CqRJArG3vItryMl7SxUA5QM1XB77iOe25m1aXEvJkUa54p6P7T62Gr0qfaEnCAeNRu33nS/16g+Znbp34jIwmmt+f3q2Uo3XV9cQgYj46ixzSjw5OPTHQSEpCHE8jtkXP80/N8jlXYLm7kTJapSWKEdbQf6J5a7Oj8Hv7Jq0QoPZmRpfa9L5PwJHd38p1JmxnVFbZMS4dQGyTwgyehf62cIDVi+5Pl3XPdQJ9FcMSTGOCXSrPGmJRHBt+o8bOuIzWMBHYqgnmHNNQ+oK/SEFhyw=
```
Comment (stemount) [inline .travis.yml]: no end of line return

File: .travis.yml (hunk 1 above)
Comment (jaffamonkey) [reply .travis.yml]: Fixed, and pushed to branch

File: .travis.yml (hunk 2 above)
Comment (jaffamonkey) [reply .travis.yml]: Fixed, and pushed to branch

**Comment by jaffamonkey** [comment]
Bit old, so closing and re-raising

**Pull request closed without merging**